core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -164,8 +167,28 @@ export class ResultPanelComponent implements OnInit {
     if (!this.showResultPanel || !this.currentOperatorId) {
       return;
     }
+    console.log("rerenderresult panel function", this.showComparisonPanel);
```

Comment:
this needs to be fixed since `previewWorkflowVersion` skips all of these. instead of just calling display differences try to handle the result update event